admin/c2cgeoportal_admin/schemas/treegroup.py

```diff
@@ -38,6 +58,7 @@ def children_validator(node, cstruct):
     for dict_ in cstruct:
         if not dict_['treeitem_id'] in [item.id for item, group in node.treeitems]:
             raise colander.Invalid(
+                node,
                 'Value {} does not exist in table {}'.
                 format(dict_['treeitem_id'], TreeItem.__tablename__))
```

Comment:
For end-user, for developer also, it should help having a less error prone message, e.g., "either value does not exits, either it exist but referencing it will cause a cycle amongst group", so to avoid looking for the non-existence of an existing value.